src/components/modeler/moveWithArrowKeys.js

```diff
@@ -0,0 +1,33 @@
+import store from '@/store';
+import PaperManager from '@/components/paperManager';
+
+export default function setUpArrowKeyEventHandlers() {
+  document.addEventListener('keydown', event => {
+    const highlightedShape = store.getters.nodeShape(store.getters.highlightedNode);
+    if (!highlightedShape) {
+      return;
+    }
+
+    highlightedShape.translate(...getPositionToMoveBy(event.key));
+  });
+}
+
+function getPositionToMoveBy(key) {
+  const moveAmount = PaperManager.gridSize / 2;
```

Comment:
I used the grid size as the base movement amount and divided it by two to allow users more fine grained control. Moving by the grid size is what happens when dragging,